mappers/tourist-attraction.js

```diff
@@ -59,7 +59,7 @@ export default function mapTouristAttractions(records, translations) {
       if (type) {
         publicG.add(sym(attractionUri), SCHEMA('keywords'), sym(type));
       } else if (type === undefined) {
-        console.error(`Cannot map location type value '${record['location_type']}' for record ${recordId}`);
+        errorLogger('location_type', record['location_type'], recordId);
```

Comment:
Pass the same field name as used to lookup the value as first argument. I know it was different before, but old code is wrong.